platform/product-management/release-plan-template.md

```diff
@@ -25,23 +57,49 @@ That's what this Release Plan Template is for! And note - there are feature togg
 - Any UX changes necessary based on the logs, or feedback on user challenges, or VA challenges? yes/no 
 - If yes, what: lorem ipsum
 
-## Phase II: unmoderated production testing
+## Phase II: unmoderated production testing (also known as Staged Rollout)
 
 ### Planning:
 - Desired date range: mm/dd/yy - mm/dd/yy
-- Desired number of unique users: x
 - How you'll make the product available in production while limiting the # of users who can find/access it: lorem ipsum
 - "Success" criteria (by the numbers): [use your KPIs to help guide this. It could be things like abondomnent rate < 20%, reported contact center calls < 2 calls, error rate < 5%, etc.]
```

Comment:
Suggest clarifying success criteria — something like .. 

- "Success" criteria (by the numbers): [user your KPIs to help guide this; it could be things like abandonment rate <20%, reported contact center calls < 2 calls, error rate < 5 %, etc. What metrics must be met for you to move to the next release phase? What metrics will you be monitoring to see if your feature has introduced an issue?]

That's pretty wordy ... maybe we can cut it back or make it look better with formatting, but I do think we need to clarify that our guidance is to set success criteria in advance, to monitor those criteria during rollout, and to only proceed with rollout if the success criteria are met.

Reply:
did some rewording/restructuring around this, lmk what you think

Reply:
Looks better! I think "confirming" _could_ cause confusion (what does it mean to confirm a metric?) but I'm fine with leaving your edits as is and trying it out.

Reply:
OK! reworded slightly once more, but feel free to come back through and change this to whatever you think sounds more understandable